### PR TITLE
Import/Export SDK and tests

### PR DESCRIPTION
**Pull request by nathannfan**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                {
+                    Location = server.Location
+                });
+                sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName2, new Database()
+                {
+                    Location = server.Location
+                });
+
+                // Get Storage container credentials
+                string testMode = HttpMockServer.GetCurrentMode().ToString();
```
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs]: what type is this originally (before you convert to string)?

File: src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                foreach(ImportExportOperationResponse operation in listOperationResponse)
+                {
+                    ImportExportOperationResponse getOperationResponse = sqlClient.ImportExportOperations.GetByServer(resourceGroup.Name, server.Name, operation.RequestId.Value);
+                    Assert.NotNull(getOperationResponse);
+                }
+            });
+        }
+
+        [Fact]
+        public void TestExportImportNewDatabaseGetOperation()
```
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs]: these 2 tests are basically the same, right? if it reduces a lot of duplication can you have these 2 test methods call one common method that gets called with 2 different modes, e.g. `TestExportImportGetOperation(bool existingDatabase)`?

**Review by jaredmoo — APPROVED** [review]

**Comment by cormacpayne** [comment]
Corresponding spec: https://github.com/Azure/azure-rest-api-specs/pull/985

@markcowl can you take a look?

**Review by cormacpayne — CHANGES_REQUESTED** [review]
@nathannfan need a version bump in [`project.json`](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/project.json#L2) as well as [`AssemblyInfo.cs`](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Properties/AssemblyInfo.cs#L25).

You will also need to update [`generate.cmd`](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/generate.cmd#L9) with Swagged used to generate this code (**this should include the commit ID as well**).

I believe you will be using this link: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/87f4b723a50b1b8714cb9c3fc5de556162348225/arm-sql/compositeSql.json but double-check to make sure that is the correct commit used.

**Review by cormacpayne — APPROVED** [review]

**Comment by cormacpayne** [comment]
LGTM when build passes

**Pull request merged**